services/web/server/src/simcore_service_webserver/projects/projects_db.py

```diff
@@ -36,7 +37,8 @@
 def _convert_to_db_names(project_document_data: Dict) -> Dict:
     converted_args = {}
     for key, value in project_document_data.items():
-        converted_args[ChangeCase.camel_to_snake(key)] = value
+        if key != 'tags': # No column for tags
```

Comment:
yeah .. we should simplify this before we have to add here too many exceptions.
I take note.